.changelog/unreleased/features/2252-nft-commitment-query-by-denom.md

```diff
@@ -0,0 +1,5 @@
+* Enhanced `GetAccountCommitments` to support querying NFT coin commitments with optional asset filter in the exchange module [#2252](https://github.com/provenance-io/provenance/issues/2252).
+* The `GetAccountCommitments` query now lets you filter commitments by a specific asset, thanks to an optional asset parameter.
+* The CLI command `SetupCmdQueryGetAccountCommitments` was updated to include a --asset flag, making it easier to filter commitments by asset directly from the command * line.
```

Comment:
Instead of `SetupCmdQueryGetAccountCommitments`, use the actual command that people would be typing, `query exchange account-commitments`. You'll also want to tweak the wording to not use the word "asset". And there's an extra `*` toward the end of the CLI entry.